deeprankcore/feature/atom.py

```diff
@@ -0,0 +1,46 @@
+from deeprankcore.models.structure import Atom
+from deeprankcore.models.graph import Graph
+from deeprankcore.models.forcefield.vanderwaals import VanderwaalsParam
+from deeprankcore.domain.forcefield import atomic_forcefield
+from deeprankcore.models.error import UnknownAtomError
+import logging
+
+
+_log = logging.getLogger(__name__)
+
+
+# these will ultimately be moved to deeprankcore.domain.feature.py and imported in this file.
+# for now, I am keeping them here to avoid merge conflicts with the feature renaming branch.
+
+# from models.structure.Atom
+FEATURE_NODE_ATOMICELEMENT = "atomic_element" # AtomicElement object
```

Comment:
I'd call it "atom_type" as agreed with Daniel (I think that it's not really the atomic element, indeed it can have only 6 values, much less than the actual number of existing atoms; I asked clarifications for this to Daniel and I'll let you know)